### DISCUSSION TITLE
Window icon

### DISCUSSION THREAD

**Discussion by refaelsh** [xmonad/xmonad#472] [Q&A] [answered]
Hi all,
Let's say I am developing an application and I want to test it in XMonad.
Specifically I wanna debug an icon window issue, the main icon window, the one that in the left top corner of a window in a Gnome/KDE/etc.
But, since there is no window decorations at all in XMonad, I can't debug the icon issue. What can be done? How can I make XMonad to at least temporary show windows decorations and/or the icon?

**Comment by liskin** [comment, marked as answer]
I don't think there's anything in xmonad or xmonad-contrib that can do this (I grepped for `_NET_WM_ICON` and got no relevant hits). You might need to temporarily run a different window manager (e.g. in a nested Xephyr instance) to see the icons and debug icon issues.

**Comment by geekosaur** [comment]
That's correct. The only kinds of icons we support are lists of `Bool`s in `Decoration` (very simple 2-color icons); you'll need to run a different window manager to see them.

**Comment by refaelsh** [comment]
I understand. Thank you all very much.
